changelog/17.0/17.0.0/summary.md

```diff
@@ -294,6 +294,29 @@ The default file can be found in `./config/init_db.sql`.
 #### <a id="vttablet-deprecated-flags"/> Deprecated Flags
 The flag `use_super_read_only` is deprecated and will be removed in a later release.
 
+Various flags that took float values as seconds have updated to take the standard duration syntax as well.
+Float-style parsing is now deprecated and will be removed in a later release.
+For example, instead of `--queryserver-config-query-pool-timeout 12.2`, use `--queryserver-config-query-pool-timeout 12s200ms`.
+Affected flags and YAML config keys:
+- `degraded_threshold`
+- `heartbeat_interval`
```

Comment:
that sounds like a breaking change, should this be under the breaking change section?

Reply:
it's not, both the old and new parsing work in 17.0. in 18.0 it'll break. see https://github.com/vitessio/vitess/pull/12920/files#diff-dfc8c56e0402c499d7e4c37432f45d2987f294ba0544dd590e5b4e1838255854R43-R58

Reply:
Okay @ajm188 !